infrastructure/pcluster-manager.yaml

```diff
@@ -259,11 +259,11 @@ Resources:
       UserPoolId: !If [ UseExistingCognito, !Ref UserPoolId, !GetAtt [ PclusterManagerCognito, Outputs.UserPoolId ]]
       PreventUserExistenceErrors: ENABLED
       RefreshTokenValidity: 7
-      AccessTokenValidity: 1
-      IdTokenValidity: 1
+      AccessTokenValidity: 10
+      IdTokenValidity: 10
       TokenValidityUnits:
-        AccessToken: "days"
-        IdToken: "days"
+        AccessToken: "minutes"
+        IdToken: "minutes"
 
```

Comment:
question: have you checked `update_infrastructure.sh` can successfully run after these changes?

Reply:
I haven't run the command so unfortunately no (but I will of course check for the outcome and fix in case)